build.nix

```diff
@@ -0,0 +1,130 @@
+let pkgs = import ./nix;
```

Comment:
Can this be moved into the `nix/` subfolder? Maybe `nix/wire-server.nix`? I wanted to do a bit of cleanup in `nix/` too, and pull in some of the `docs/` folder and the plumbing there.

Essentially have `nix/default.nix` return more than just `pkgs`, but an attrset, which could contain `nixpkgs`, and a `wire-server` sub-namespace.

Reply:
Moved it to `nix/wire-server.nix`. The `devEnv` is not `wireServer.devShell` (because it is a nix-shell now).